### PR TITLE
Provide new default dev env for tt and cts1.

### PR DESCRIPTION
**Pull request by KineticTheory**
### Background

* Brand new spack builds for TPLs on trinitite and snow are now available.  This PR updates the default dev environment (via `.bashrc` files) to use the new installations.
* The new TPL setup has the following features:

*Advantages:*
  * cts-1 modules are hierarchical (like ccs-net and rzansel)
  * cts-1 modules are much faster than HPC’s system
  * Very easy for user_contrib developer to add new installs.  No manual modulefile manipulation (except for new compilers).
  * ats-1 modules use descriptive names.

*Disadvantages*
* For cts-1, it takes extra work to get to HPC provided modules that haven’t been migrated to user_contrib.
* I still need to work with the PRE team to convert the ‘user_contrib’ modulefile into a directory (/usr/projects/draco/Modules/snow/user_contrib/ will be our temporary replacement for the official user_contrib module).
* For tt, there are long lists of available modules.
 
### Purpose of Pull Request

* In person review of design and mechanics is scheduled for 8/24.
* [Fixes Redmine Issue #3](https://rtt.lanl.gov/redmine/issues/3)

### Description of changes

* I'm leaving a lot of the old code in-place but commented out.

### Status

* Reference:
  * [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [CDash Status](https://rtt.lanl.gov/cdash/index.php?project=Draco&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-pr)
* Checks
  * [ ] Travis/Appveyor CI checks pass
  * [ ] Code coverage does not decrease
  * [ ] [Valgrind test passes](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [ ] [Toss3 checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [ ] [Trinitite checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [ ] Code reviewed/approved, sufficient DbC checks, testing, documentation
  * [ ] fstools
  * [ ] LAP env check (`mpiifort` missing)

### CODE REVIEW

**Comment by KineticTheory** [comment]
@jhchang-lanl Do you have time to check this out manually and test it for me?

File: environment/bashrc/.bashrc_cray [hunk 1]
```diff
@@ -82,51 +78,14 @@ arch=`uname -m`
...
 sysname=`/usr/projects/hpcsoft/utilities/bin/sys_name`
-#if test -d $HOME/modulefiles/$sysname; then
-#  export ucmf=$HOME/modulefiles
-#  module use --append ~/modulefiles/$sysname
-#else
-  module load user_contrib
-#fi
-module load friendly-testing
-if [[ ${SLURM_JOB_PARTITION} == "knl" ]]; then
-  module swap craype-haswell craype-mic-knl
```
Comment (david8dixon) [inline environment/bashrc/.bashrc_cray:93]: so going forward do we load `draco/intel19` and then do the swap by hand?

Comment (KineticTheory) [reply environment/bashrc/.bashrc_cray:93]: That is the idea.  We load the 'production' set of tools by default.  Anyone can swap out individual components as they see fit.  We can create other _super modules_ for alternate sets (e.g. intel-17 or gcc-9).

**Comment by david8dixon** [comment]
This looks good - I will test it now.

**Pull request merged**